core/embed/rust/src/ui/constant.rs

```diff
@@ -1,7 +1,13 @@
 //! Reexporting the `constant` module according to the
 //! current feature (Trezor model)
 
-#[cfg(all(feature = "model_tr", not(feature = "model_tt")))]
+#[cfg(feature = "model_mercury")]
+pub use super::model_mercury::constant::*;
+#[cfg(all(
+    feature = "model_tr",
+    not(feature = "model_tt"),
+    not(feature = "model_mercury")
+))]
 pub use super::model_tr::constant::*;
-#[cfg(feature = "model_tt")]
+#[cfg(all(feature = "model_tt", not(feature = "model_mercury")))]
 pub use super::model_tt::constant::*;
```

Comment:
The model selector needs to be spelled the same way every time.
When all features are enabled, this selects model_mercury for constants, but model_tt for ui-features.

Reply:
uh, good catch, 9f4dc34cb76dfeb25f98d31258977bad3d35e112